api/all/src/main/java/io/opentelemetry/api/metrics/ObservableDoubleMeasurement.java

```diff
@@ -12,15 +12,38 @@
   /**
    * Records a measurement.
    *
-   * @param value The measurement amount. MUST be non-negative.
+   * @param value The measurement amount.
+   * @deprecated Use {@link #record(double)}.
    */
-  void observe(double value);
+  @Deprecated
```

Comment:
Include a message about using `record(value)` instead?

Reply:
Yeah it's there in the javadoc, this is the annotation 😛